packages/data-row-edit/vue.config.js

```diff
@@ -0,0 +1,40 @@
+const BannerPlugin = require('webpack').BannerPlugin
+const pkgVersion = require('./package.json').version
+const pkgName = require('./package.json').name
+
+const now = new Date()
+const buildDate = now.toUTCString()
+const bannerText = `package-name: ${pkgName}
+package-version: ${pkgVersion}
+build-date: ${buildDate}`
+
+const target = 'https://master.dev.molgenis.org'
+module.exports = {
+  filenameHashing: false,
+  outputDir: 'dist',
+  publicPath: process.env.NODE_ENV === 'production'
+    ? '/@molgenis-ui/data-row-edit/dist/'
+    : '/',
+  'devServer': {
+    // In CI mode, Safari cannot contact "localhost", so as a workaround, run the dev server using the jenkins agent pod dns instead.
+    host: process.env.JENKINS_AGENT_NAME || 'localhost',
+    proxy: process.env.NODE_ENV === 'production' ? undefined : {
+      '/login': { target },
+      '/api': { target }
+    }  
+  },
+  configureWebpack: config => {
+    config.optimization = {
```

Comment:
Die optimalisatie waarschuwing is wat lelijk. Ik zou die zelf uitzetten of op korte termijn die waarschuwingen wegwerken. Uitzetten kan met:
`config.performance = { hints: false }`